spec/models/distribution_spec.rb

```diff
@@ -43,6 +43,13 @@
       d.line_items << build(:line_item, item: item_missing)
       expect(d).not_to be_valid
     end
+
+    it "ensures that the issued at is no earlier than 2000" do
+      d = build(:distribution, issued_at: '1999-12-31')
+      expect(d).not_to be_valid
+    end
```

Comment:
Really this should be in a spec for the IssuedAt concern. But if we don't have one, carry on :)

Reply:
Carrying on.